etc/config/fortran.amazon.properties

```diff
@@ -0,0 +1,114 @@
+compiler.g472.exe=/opt/compiler-explorer/gcc-4.7.2/bin/gfortran
+compiler.g472.name=x86-64 gfortran 4.7.2
+compiler.g473.exe=/opt/compiler-explorer/gcc-4.7.3/bin/gfortran
+compiler.g473.name=x86-64 gfortran 4.7.3
+compiler.g473.alias=/usr/bin/gfortran-4.7
+compiler.g474.exe=/opt/compiler-explorer/gcc-4.7.4/bin/gfortran
+compiler.g474.name=x86-64 gfortran 4.7.4
+compiler.g481.exe=/opt/compiler-explorer/gcc-4.8.1/bin/gfortran
+compiler.g481.name=x86-64 gfortran 4.8.1
+compiler.g482.exe=/opt/compiler-explorer/gcc-4.8.2/bin/gfortran
+compiler.g482.name=x86-64 gfortran 4.8.2
+compiler.g483.exe=/opt/compiler-explorer/gcc-4.8.3/bin/gfortran
+compiler.g483.name=x86-64 gfortran 4.8.3
+compiler.g484.exe=/opt/compiler-explorer/gcc-4.8.4/bin/gfortran
+compiler.g484.name=x86-64 gfortran 4.8.4
+compiler.g485.exe=/opt/compiler-explorer/gcc-4.8.5/bin/gfortran
+compiler.g485.name=x86-64 gfortran 4.8.5
+compiler.g485.alias=/usr/bin/gfortran-4.8
+compiler.g490.exe=/opt/compiler-explorer/gcc-4.9.0/bin/gfortran
+compiler.g490.name=x86-64 gfortran 4.9.0
+compiler.g490.alias=/opt/gcc-4.9.0/bin/gfortran
+compiler.g491.exe=/opt/compiler-explorer/gcc-4.9.1/bin/gfortran
+compiler.g491.name=x86-64 gfortran 4.9.1
+compiler.g492.exe=/opt/compiler-explorer/gcc-4.9.2/bin/gfortran
+compiler.g492.name=x86-64 gfortran 4.9.2
+compiler.g493.exe=/opt/compiler-explorer/gcc-4.9.3/bin/gfortran
+compiler.g493.name=x86-64 gfortran 4.9.3
+compiler.g494.exe=/opt/compiler-explorer/gcc-4.9.4/bin/gfortran
+compiler.g494.name=x86-64 gfortran 4.9.4
+compiler.g510.exe=/opt/compiler-explorer/gcc-5.1.0/bin/gfortran
+compiler.g510.name=x86-64 gfortran 5.1
+compiler.g520.exe=/opt/compiler-explorer/gcc-5.2.0/bin/gfortran
+compiler.g520.name=x86-64 gfortran 5.2
+compiler.g530.exe=/opt/compiler-explorer/gcc-5.3.0/bin/gfortran
+compiler.g530.name=x86-64 gfortran 5.3
+compiler.g540.exe=/opt/compiler-explorer/gcc-5.4.0/bin/gfortran
+compiler.g540.name=x86-64 gfortran 5.4
+compiler.g550.exe=/opt/compiler-explorer/gcc-5.5.0/bin/gfortran
+compiler.g550.name=x86-64 gfortran 5.5
+compiler.g6.exe=/opt/compiler-explorer/gcc-6.1.0/bin/gfortran
+compiler.g6.name=x86-64 gfortran 6.1
+compiler.g62.exe=/opt/compiler-explorer/gcc-6.2.0/bin/gfortran
+compiler.g62.name=x86-64 gfortran 6.2
+compiler.g63.exe=/opt/compiler-explorer/gcc-6.3.0/bin/gfortran
+compiler.g63.name=x86-64 gfortran 6.3
+compiler.g71.exe=/opt/compiler-explorer/gcc-7.1.0/bin/gfortran
+compiler.g71.name=x86-64 gfortran 7.1
+compiler.g72.exe=/opt/compiler-explorer/gcc-7.2.0/bin/gfortran
+compiler.g72.name=x86-64 gfortran 7.2
+compiler.g73.exe=/opt/compiler-explorer/gcc-7.3.0/bin/gfortran
+compiler.g73.name=x86-64 gfortran 7.3
+compiler.g81.exe=/opt/compiler-explorer/gcc-8.1.0/bin/gfortran
+compiler.g81.name=x86-64 gfortran 8.1
+compiler.gsnapshot.exe=/opt/compiler-explorer/gcc-snapshot/bin/gfortran
+compiler.gsnapshot.name=x86-64 gfortran (trunk)
+compiler.gsnapshot.alias=g7snapshot
+
+###############################
+# GCC for ppc
```

Comment:
Could you separate PowerPC to its own group, which derives from this one? (Like what's done with PowerPC in C++)
Thanks!

Reply:
I believe I copied what I found in `c++.amazon.properties` but found it odd to combine PPC and ARM the way it's done right now. Will try and change it.